### PR TITLE
MSM - Moved file copy and sanity check to easyblock

### PR DESCRIPTION
**Pull request by vanzod**
This allows a cleaner easyconfig file by moving the definition of the files to copy and the sanity check to the easyblock.

### CODE REVIEW

File: easybuild/easyblocks/m/msm.py [hunk 1]
```diff
@@ -98,6 +110,22 @@ def build_step(self):
...
         """Override installdir creation"""
+
         self.log.warning("Not removing installation directory %s" % self.installdir)
         self.cfg['keeppreviousinstall'] = True
         super(EB_MSM, self).make_installdir()
+
+    def install_step(self):
+        """Define files to be copied at installation."""
+
+        self.cfg['files_to_copy'] = [(['src/MSM/%s' % f for f in self.binfiles], 'bin')]
```
Comment (boegel) [inline easybuild/easyblocks/m/msm.py]: please use `os.path.join`:

```python
self.cfg['files_to_copy'] = [([os.path.join('src', 'MSM', f) for f in self.binfiles], 'bin')]
```

Also, aren't `doc` & co missing? How will those get installed?

File: easybuild/easyblocks/m/msm.py [hunk 2]
```diff
@@ -98,6 +110,22 @@ def build_step(self):
...
+        super(EB_MSM, self).install_step()
+
+    def sanity_check_step(self):
+        """Custom sanity check for MSM."""
+
+        custom_paths = {
+            'files': ['bin/%s' % f for f in self.binfiles],
```
Comment (boegel) [inline easybuild/easyblocks/m/msm.py]: here too:

```python
[os.path.join('bin', f) for f in self.binfiles]

File: easybuild/easyblocks/m/msm.py (hunk 1 above)
Comment (vanzod) [reply easybuild/easyblocks/m/msm.py]: @boegel `doc` are installed automatically by `make install`. As a matter of fact the `doc` directory is already checked in `sanity_check_step`.

Comment (boegel) [reply easybuild/easyblocks/m/msm.py]: Wait, `make install` installs the docs, but not the binaries?! 🤦‍♂

Comment (vanzod) [reply easybuild/easyblocks/m/msm.py]: Yup

**Pull request merged**